src/components/ApiReference/ScrollRouter.js

```diff
@@ -60,7 +60,11 @@ export const ScrollRouter = ({ children, initialActive = "" }) => {
       // before, update the route.
       if (newActiveNode && newActiveNode !== activeNode) {
         setActiveNode(newActiveNode);
-        window.history.replaceState(null, null, routeMap.get(newActiveNode));
+        window.history.replaceState(
+          null,
+          null,
+          `${routeMap.get(newActiveNode)}${window.location.search}`,
```

Comment:
should we be concerned about google indexing the url w the query param? not sure if they do that. 

for ex: https://developers.stellar.org/api/introduction/pagination/?javascript=false

Reply:
hmmm this is an excellent question

Reply:
I _believe_ this will work fine, because we have a canonical URL set. Google appears to have retired all the tools it used to expose for manually mucking with the URL params in their webmaster tools